QtlTodoList/Manager/FirebaseManager.swift

```diff
@@ -71,4 +78,30 @@ extension FirebaseManager {
             print(error.localizedDescription)
         }
     }
+    
+    /// 画像アップロード
+    func todoImageUpload(image: UIImage) async -> URL? {
+        let reference = storage.reference().child("image/image\(UUID().uuidString).jpg")
+        guard let resizedImage = image.resizeUIImage(width: image.size.width, height: image.size.height) else {
+            return nil
+        }
+        guard let data = resizedImage.jpegData(compressionQuality: 0.2) else { return nil }
+        let metaData = StorageMetadata()
+        metaData.contentType = "image/jpg"
+        do {
+            let _ = try await reference.putDataAsync(data)
```

Comment:
```suggestion
            _ = try await reference.putDataAsync(data)
```

[nits] 不要であればletはいらないかなと

Reply:
修正しました！
7f0c9e60ed34f9ecbbad1cde0b31823a1438716d